src/transforms/dedupe.rs

```diff
@@ -174,11 +176,20 @@ impl Dedupe {
     }
 
     fn transform_one(&mut self, event: Event) -> Option<Event> {
+        emit!(EventsReceived {
+            count: 1,
+            byte_size: event.size_of()
+        });
         let cache_entry = build_cache_entry(&event, &self.fields);
         if self.cache.put(cache_entry, true).is_some() {
-            emit!(DedupeEventDiscarded { event });
+            emit!(DedupeEventsDropped { events: [event] });
             None
         } else {
+            emit!(EventsSent {
```

Comment:
Do we need to emit `EventsSent`? I thought it was emitted by the topology layer for transforms.

Reply:
Same with `EventsReceived`.

Reply:
Correct on both counts.

Reply:
That's correct. We discussed this during Friday's check-in. Pablo is going to update all relevant PRs to (1) remove the duplicate events being emitted, and (2) update the test harness to actually run the topology layer, since that causes the tests to miss these events being fired, which in turn caused Pablo to add the events to the individual transforms.